packages/react-renderer-demo/src/app/src/doc-components/pf4-select.md

```diff
@@ -1,3 +1,11 @@
+# Menu portaling
+
+In order to show menu properly in dialogs like Modals, you can use `menuIsPortal`. This will append portal to body to render menu.
```

Comment:
`modals` with small m?

`This will append portal to body to render menu.` Is this sentence right?